src/Integration.Vsix/Commands/ConnectedModeMenu/ConnectedModeMenu.vsct

```diff
@@ -4,29 +4,35 @@
   <Commands package="guidVsPackagePkgString">
     <Groups>
       <Group guid="ConnectedModeMenuCmdSet" id="ConnectedModeGroup">
-        <Parent guid="SonarLintCmdSet" id="ConnectedModeMenu"/>
+        <Parent guid="SonarLintCmdSet" id="ConnectedModeMenu" />
       </Group>
     </Groups>
 
     <Buttons>
       <Button guid="ConnectedModeMenuCmdSet" id="ManageServerConnectionsCommand">
-        <Parent guid="ConnectedModeMenuCmdSet" id="ConnectedModeGroup"/>
+        <Parent guid="ConnectedModeMenuCmdSet" id="ConnectedModeGroup" />
         <Strings>
           <ButtonText>Bind to SonarQube or SonarCloud...</ButtonText>
         </Strings>
       </Button>
+      <Button guid="ConnectedModeMenuCmdSet" id="SaveSharedConnectionCommand">
+        <Parent guid="ConnectedModeMenuCmdSet" id="ConnectedModeGroup" />
+        <Strings>
+          <ButtonText>Save Shared Connection Binding</ButtonText>
```

Comment:
This sounds off. Generate Shared Binding Configuration? 'Connection' definitely sounds weird